README.md

```diff
@@ -20,3 +24,18 @@ Discuss features and ask questions on
 ## License
 
 [MIT](LICENSE).
+
+## Module Long Term Support Policy
+
+This module adopts the [Module Long Term Support (LTS)](http://github.com/CloudNativeJS/ModuleLTS) policy, with the following End Of Life (EOL) dates:
+
+| Version | Status          | Published | EOL      |
+| ------- | --------------- | --------- | -------- |
+| 6.x     | Active LTS      | Nov 2018  | Dec 2019 |
```

Comment:
please see similar comments in here: https://github.com/strongloop/loopback-sdk-angular-cli/pull/79/files#r235835697
For 6.x, following the Node.js 10 LTS schedule, it should be April 2021